mne/time_frequency/tfr.py

```diff
@@ -1065,7 +1065,7 @@ def save(self, fname, overwrite=False):
 def _prepare_write_tfr(tfr, condition):
     """Aux function"""
     return (condition, dict(times=tfr.times, freqs=tfr.freqs,
-                            data=tfr.data, info=tfr.info, nave=tfr.nave,
+                            data=tfr.data, info=tfr.info._store, nave=tfr.nave,
```

Comment:
using `info._store` should be a bit faster (won't do any iterating over the object to construct a new dict) so I prefer it